daemon/systemd/sailjaild.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Application Permission Management Daemon
+Requires=dbus.service
+After=dbus.socket pre-user-session.target
+
+[Service]
+Type=notify
+ExecStart=/usr/bin/sailjaild --systemd
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
How we handle this [Install] thing feels a bit iffy to me, but...  in any case WantedBy just includes a service in the pool of things that ought to be started. When it eligible to be started depends on After etc - right? (And that part probably needs a second look).

That said, this defines the usual case: when we boot into user mode, the service should be started. If os update has/gains need for the service, it should/can add suitable requires/after/wants things in that context, or?

Reply:
I'm not sure [Install] section is even relevant. If the service is enabled by symlink in packaging then it just bypasses these instructions.

I would argue that we don't need sailjaild before user session is starting. There are not plans to use it outside user session and it's not used for services, only applications (and perhaps services provided by applications but that's slightly different). We can't even prompt for permissions unless there is a working user session running. Thus multi-user.target is perfectly fine.

Wrt. os updates, if we need to start this in that case, then we just add similar symlink to its target. I don't think there is any earlier common target that would make sense for this service anyway.

That After pre-user-session.target is probably not that useful I think. Before systemd-user-sessions.service might be but not necessarily. If some other service wants to listen to signals from sailjaild.service, then it can say that it needs to start before it and systemd should just take care of that.

Reply:
I dropped that After=pre-user-session.target. Plus assuming that if there is some special things update mode needs it will be handled separately.

Reply:
I told exactly about the OS update case (or vendor-data installation case) when the applications are installed at early system target, and the daemon could be needed.